tests/subsys/llext/CMakeLists.txt

```diff
@@ -71,10 +71,15 @@ if (CONFIG_LLEXT_TYPE_ELF_RELOCATABLE AND NOT CONFIG_ARM AND NOT CONFIG_RISCV)
   # Manually fix the pre_located extension's text address at a multiple of 4
   get_target_property(pre_located_target pre_located_ext lib_target)
   get_target_property(pre_located_file pre_located_ext pkg_input)
+
+  get_target_property(zephyr_flags zephyr_interface INTERFACE_COMPILE_OPTIONS)
+  string(REGEX MATCH "-mcpu=[^$>-\;]+" cpu_flag "${zephyr_flags}")
```

Comment:
surely there's a better way to get a compiler flag like this than a regex, @tejlmand any hints?

Reply:
I think I found a better way than a string regex to get the flag.

```
-mcpu=${GCC_M_CPU}
```